ui/src/map/layers/geometry/geometry-switch-layer.ts

```diff
@@ -31,74 +31,74 @@ export function createGeometrySwitchLayer(
     const vectorSource = existingOlLayer?.getSource() || new VectorSource();
     const layer = existingOlLayer || new VectorLayer({ source: vectorSource });
 
-    const visibleSwitches = manuallySetPlan
-        ? manuallySetPlan.switches.map((s) => s.sourceId)
-        : selection.visiblePlans.flatMap((p) => p.switches);
+    const shownSwitches = () => {
+        if (resolution <= Limits.SWITCH_SHOW) {
+            return manuallySetPlan
+                ? manuallySetPlan.switches.map((s) => s.sourceId)
+                : selection.visiblePlans.flatMap((p) => p.switches);
+        } else {
+            return selection.selectedItems.geometrySwitchIds.map((s) => s.geometryId);
+        }
+    };
 
-    let inFlight = false;
-    if (resolution <= Limits.SWITCH_SHOW) {
```

Comment:
Tästä eteenpäin ei ole oikeasti mitään muutoksia kuin tämän iffiin ehdon ja lopun else-haaran poisto. Sisennykset (ja sitä kautta rivitykset) muuttuivat tämän takia, niin git ei vähemmän yllättäen tajua näitä oikein